tests/chainer_tests/functions_tests/loss_tests/test_huber_loss.py

```diff
@@ -18,13 +18,13 @@
       'double_backward_options': {'eps': 1e-1, 'rtol': 1e0, 'atol': 1e0}},
      {'dtype': numpy.float32,
       'forward_options': {},
-      'backward_options': {'eps': 1e-2, 'rtol': 1e-2, 'atol': 1e-2},
-      'double_backward_options': {'eps': 1e-2, 'rtol': 1e-1, 'atol': 1e-1}},
+      'backward_options': {'eps': 1e-3, 'rtol': 1e-2, 'atol': 1e-2},
```

Comment:
FYI, the `'eps'` setting could be just removed, because `1e-3` is the default of `check_backward` and `check_double_backward`.

Reply:
If we fix [this](https://github.com/chainer/chainer/pull/6924#discussion_r275716750), we cannot use the default `eps`.